sql/driver/driver_test.go

```diff
@@ -31,7 +32,7 @@ var isError = testutils.IsError
 
 func setup(t *testing.T) (*server.TestServer, *sql.DB) {
 	s := server.StartTestServer(nil)
-	db, err := sql.Open("cockroach", "https://root@"+s.ServingAddr()+"?certs=test_certs")
+	db, err := sql.Open("cockroach", "https://"+security.RootUser+"@"+s.ServingAddr()+"?certs=test_certs")
```

Comment:
So you're pessimizing the common case of reading the code in favor of making it easier to potentially change the named constant which is unlikely to ever happen in this case.